CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.0.6
+
+### [0.0.6](https://github.com/openfga/syntax-transformer/compare/v0.0.5...v0.0.6) (2022-10-11)
+
+#### Changes
+- fix(check-dsl): allow same relation in computedUserset of from clause (openfga/syntax-transformer#70)
+- chore(deps): bump dependencies
```

Comment:
I don't think we updated dependencies 🤔

Changelist: https://github.com/openfga/syntax-transformer/compare/v0.0.5...main

Reply:
False alarm - they have been updated in this PR